CHANGES.txt

```diff
@@ -8,6 +8,8 @@ Unreleased
 Changes
 -------
 
+ - Added support for HTTP password authentication.
```

Comment:
I would only say: Added support for password authentication.
HTTP is implementation detail, maybe users are not even aware that crate uses HTTP protocol.